{"name": "test_app_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestAppLogin' object has no attribute 'login'", "trace": "self = <scripts.test05_app_login.TestAppLogin object at 0x0000000003C69CC0>, phone = '13812345678', code = '246810'\n\n    def test_app_login(self, phone='13812345678', code='246810'):\n        # 调用app登录业务方法\n>       self.login.page_app_login(phone, code)\nE       AttributeError: 'TestAppLogin' object has no attribute 'login'\n\nscripts\\test05_app_login.py:36: AttributeError"}, "start": 1596673044791, "stop": 1596673044791, "uuid": "01e0ed68-b1bc-491b-889e-9095e253d4b0", "historyId": "63e781105a8add2ee581dba9242d20e2", "testCaseId": "ec2933d9e11d0cbb035e36410d90b36e", "fullName": "scripts.test05_app_login.TestAppLogin#test_app_login", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test05_app_login"}, {"name": "subSuite", "value": "TestAppLogin"}, {"name": "host", "value": "XC003ZBOHM7MFW8"}, {"name": "thread", "value": "9100-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test05_app_login"}]}